games-simulation/thepowdertoy/thepowdertoy-91.5.330.recipe

```diff
@@ -0,0 +1,71 @@
+SUMMARY="The Powder Toy - A free physics sandbox game"
+DESCRIPTION="The Powder Toy is a free physics sandbox game,\
+which simulates air pressure and velocity, heat, gravity and \
+a countless number of interactions between different substances! \
+The game provides you with various building materials, liquids, gases \
+and electronic components which can be used to construct complex machines,\
+guns, bombs, realistic terrains and almost anything else."
+HOMEPAGE="http://powdertoy.co.uk"
+COPYRIGHT="2008 - 2010 Stanislaw Skowronek.
```

Comment:
No spaces before/after "-" and no need for the "." at the end. So it's "2008-2010 Stanislaw Skowronek"